ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# Orbit Web
+
+The current implementation status, immediate-term plans, and future goals of this repository.
+
+We welcome feature requests as edits to the "[Backlog](#backlog)" section below. Suggest something by opening a pull request editing this file.
+
+## Build Status
+
+TBA
+
+## Timeline
+
+TBA
+
+## Roadmap
+
+Our short-to-medium-term roadmap items, in order of descending priority.
+
+### _Example Backlog_
+
+| Status  | Feature                         | Owner                                    | Details                                                     |
+| :------ | :------------------------------ | :--------------------------------------- | :---------------------------------------------------------- |
+| &#9745; | example thing which is done     | [@haadcode](https://github.com/haadcode) | more detailed explanation about the thing which is done     |
+| &#9744; | example thing which is not done | [@haadcode](https://github.com/haadcode) | more detailed explanation about the thing which is not done |
+
+### Backlog
+
+| Status  | Feature               | Owner                                    | Details                                                                                                                       |
+| :------ | :-------------------- | :--------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------- |
+| &#9745; | simplify startup      | [@latenssi](https://github.com/latenssi) | start the network automatically                                                                                               |
+| &#9744; | fix build             | [@latenssi](https://github.com/latenssi) | currently build is not outputting a _standalone_ working web app, some paths are incorrect etc.                               |
```

Comment:
&#9744; appears (on my computer) as a blank square (similar to "no-unicode support" square). Is that intentional? Alternatively, we could also use emojis for the checkboxes.

Reply:
That is intentional. I am not sure about using emojis since their rendering on markdown interpreters other than Githubs is questionable.

Reply:
Ok 👍